libs/shared/src/types/organization/index.ts

```diff
@@ -6,3 +8,8 @@ export enum ApiServiceLevelTypeEnum {
   // TODO: NV-3067 - Remove unlimited tier once all organizations have a service level
   UNLIMITED = 'unlimited',
 }
+
+export type IServiceApiRateLimits = Record<ApiServiceLevelTypeEnum, IApiRateLimits>;
+
+export type ApiRateLimitEnvVarFormat =
+  Uppercase<`API_RATE_LIMIT_${ApiServiceLevelTypeEnum}_${ApiRateLimitCategoryTypeEnum}`>;
```

Comment:
Template type literal describing the required key format for an environment variable.